Postinstall.js does not work on path with spaces

# This is a Bug Report

## Description

For bug reports:
* What went wrong?

When running `npm install serverless` within a path with spaces, postinstall.js fails with following error:

```
> serverless@1.23.0 postinstall /path/with some spaces/serverless
> node ./scripts/postinstall.js

module.js:473
      throw err;
      ^

Error: Cannot find module '/path/with'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:453:25)
    at Function.Module.runMain (module.js:665:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:607:3
module.js:473
      throw err;
      ^

Error: Cannot find module '/path/with'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:453:25)
    at Function.Module.runMain (module.js:665:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:607:3
npm WARN seduction-evergreen@0.1.0 No description
npm WARN seduction-evergreen@0.1.0 No repository field.
npm WARN seduction-evergreen@0.1.0 No license field.

```

* What did you expect should have happened?

Postinstall.js to run without errors.


## Additional Data

* ***Serverless Framework Version you're using***: 1.23.0
* ***Operating System***: Ubuntu subsystem on Windows 10

--- a/scripts/postinstall.js
+++ b/scripts/postinstall.js
@@ -27,12 +27,12 @@ function setupAutocomplete() {
     const tabtabCliPath = path.join(tabtabPath, 'src', 'cli.js');
 
     try {
-      execSync(`node ${tabtabCliPath} install --name serverless --auto`);
-      execSync(`node ${tabtabCliPath} install --name sls --auto`);
+      execSync(`node "${tabtabCliPath}" install --name serverless --auto`);
+      execSync(`node "${tabtabCliPath}" install --name sls --auto`);
       return resolve();
     } catch (error) {
-      execSync(`node ${tabtabCliPath} install --name serverless --stdout`);
-      execSync(`node ${tabtabCliPath} install --name sls --stdout`);
+      execSync(`node "${tabtabCliPath}" install --name serverless --stdout`);
+      execSync(`node "${tabtabCliPath}" install --name sls --stdout`);
       console.log('Could not auto-install serverless autocomplete script.');
       console.log('Please copy / paste the script above into your shell.');
       return reject(error);

